netket/utils/struct/dataclass.py

```diff
@@ -250,6 +250,21 @@ def process_cached_properties(clz, globals={}):
     setattr(clz, _CACHES, cached_props)
 
 
+def purge_cache_fields(clz):
+    """Removes the cache fields generated by netket dataclass
+    from the dataclass mechanism.
+    """
+    flds = getattr(clz, _FIELDS, None)
+    if flds is not None:
+        caches = getattr(clz, _CACHES)
+        for name, cp in caches.items():
+            cname = _cache_name(name)
+            if cname in flds:
+                flds.pop(cname)
+
+    setattr(clz, _CACHES, cached_props)
+
+
 def purge_cache_fields(clz):
```

Comment:
I think git is confused. Because that commit is already on master.
Best thing is probably to rebase.

Reply:
It's actually duplicated in this PR (probably because of a merge at some point). One of the versions needs to be removed here. (Probably the one that is not on `master`.)